Remove mention of `deployment_tools` (#1593)
grafana/mimir: operations/mimir-mixin/docs/playbooks.md

1. Look for partial blocks in the logs. Example Loki query: `{cluster="<cluster>",namespace="<namespace>",container="compactor"} |= "skipped partial block"`
1. Pick a block and note its ID (`block` field in log entry) and tenant ID (`org_id` in log entry)

> 1. Find the bucket used by the Mimir cell

Reviewer: ```suggestion
1. Find the bucket used by the Mimir cell (eg. checking the configured `blocks_storage_bucket_name` when using jsonnet)
```

> 1. Find the bucket used by the Mimir cell (eg. checking the configured `blocks_storage_bucket_name` when using jsonnet)

Reviewer 2: ```suggestion
1. Find the bucket used by the Mimir cell, such as checking the configured `blocks_storage_bucket_name` if you are using Jsonnet.
```
